neon_utils/skills/neon_skill.py

```diff
@@ -94,6 +99,10 @@ def __init__(self, name=None, bus=None, use_settings=True):
             LOG.error(e)
             self.language_config, self.language_detector, self.translator = None, None, None
 
+    def initialize(self):
+        # schedule an event to load the cache on disk every 24 hours
+        self.schedule_event(self._reset_cache, CACHE_TIME_OFFSET, name="neon.load_cache_on_disk")
```

Comment:
Is this a read or a write? Can the name be changed to read/write/sync? Comment should reference CACHE_TIME_OFFSET